docs/examples/word_embedding/utils.py

```diff
@@ -0,0 +1,207 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=global-variable-undefined,wrong-import-position
+"""Utils for embedding example.
+
+This is based on train_fasttext.py in the scripts/word_embedding folder.
+
+"""
+import itertools
+import math
+import os
+
+import mxnet as mx
+import numpy as np
+
+import gluonnlp as nlp
+from gluonnlp.base import numba_jitclass, numba_types
+
+os.environ['MXNET_GPU_MEM_POOL_TYPE'] = 'Round'
```

Comment:
this might need to happen before importing mxnet. also, would it be better to let the users set this?

Reply:
It shouldn't matter for the docs/examples code.The statement appears here due to copy paste. Should be resolved by sharing the utils.py code with the scripts folder once symlink is supported.